syntax = "proto3";

package greet;

import "proto_2/dependency.proto";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply);
}

message HelloRequest {
  string name = 1;
  dependency.UserInfo user_info = 2;
}

message HelloReply {
  string message = 1;
}
